releases/v20.1.1.md

```diff
@@ -0,0 +1,229 @@
+- Fixed a rare bug causing a range to deadlock and all the writes to the respective range to timeout. [#48303][#48303]
+- Fixed a long-standing bug where HTTP requests would start to fail with error 503 "`transport: authentication handshake failed: io: read/write on closed pipe`" and never become possible again until the node is restarted. [#48456][#48456]
+- Fixed a bug where vectorized queries on composite datatypes could sometimes return invalid data. [#48463][#48463]
+- When processing `--join`, invalid SRV records with port number `0` are now properly ignored. [#48527][#48527]
+- Fixed a bug where `SHOW STATISTICS USING JSON` contained incorrect single quotes for strings with spaces inside histograms. [#48544][#48544]
+- Fixed a bug where the two settings `kv.range_split.by_load_enabled` and `kv.range_split.load_qps_threshold` were incorrectly marked as non-public in the output of `SHOW CLUSTER SETTINGS`. [#48585][#48585]
+- You can no longer drop databases that contain tables which are currently offline due to `IMPORT` or `RESTORE`. Previously dropping a database in this state could lead to a corrupted schema which prevented running backups. [#48606][#48606]
+- Fixed a bug preventing timestamps from being closed which could result in failed follower reads or failure to observe resolved timestamps in changefeeds. [#48682][#48682]
+- Fixed `debug encryption-status` and the Admin UI display of encryption status when using Pebble. [#47995][#47995]
+- CockroachDB now deletes the partially imported data after an `IMPORT` fails or is canceled. [#48605][#48605]
+- Fixed a bug where the `SHOW CREATE` statement would sometimes show a partitioning step for an index that has been dropped. [#48768][#48768]
+- Re-allowed `diagnostics.forced_sql_stat_reset.interval`, `diagnostics.sql_stat_reset.interval` and `external.graphite.interval` to set to their maximum values (24hr, 24hr and 15min respectively). [#48760][#48760]
+- Fixed a bug where CockroachDB could encounter an internal error when a query with `LEFT SEMI` or `LEFT ANTI` join was performed via the vectorized execution engine. This is likely to occur only with `vectorize=on` setting. [#48751][#48751]
+- Fixed a bug where running `cockroach dump` on tables with interleaved primary keys would erroneously include an extra `CREATE UNIQUE INDEX "primary" ... INTERLEAVE IN PARENT` statement in the dump output. This made it impossible to reimport dumped data without manual editing. [#48776][#48776]
+- Fixed a bug where running `cockroach dump` on a table with collated strings would omit the collation clause for the data insertion statements. [#48832][#48832]
+- Fixed a bug where the migration for ongoing schema change jobs would cause the node to panic with an "index out of bounds" error upon encountering a malformed table descriptor with no schema change mutation corresponding to the job to be migrated. [#48838][#48838]
```

Comment:
Is this the schema change crash-loop bug that I saw folks talking about in Slack?  If it is, I think we should move it higher in the list and mention that the node will panic again if it is restarted.  But I'm not sure if this is that bug.